src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt

```diff
@@ -722,7 +734,7 @@ class LicenceService(
         it.copy(additionalCondition = condition, uploadDetailId = uploadDetail.id)
       }
 
-      condition.copy(additionalConditionData = updatedAdditionalConditionData, additionalConditionUploadSummary = updatedAdditionalConditionUploadSummary)
+      condition.copy(additionalConditionData = updatedAdditionalConditionData as MutableList<AdditionalConditionData>, additionalConditionUploadSummary = updatedAdditionalConditionUploadSummary)
```

Comment:
` as MutableList<AdditionalConditionData>,` shouldn't be needed

Reply:
That was mistake . I should have taken it out . Thank you